.pre-commit-config.yaml

```diff
@@ -101,152 +97,135 @@ repos:
   # Local checks for PEP headers and more
   - repo: local
     hooks:
-      - id: check-no-tabs
-        name: "Check tabs not used in PEPs"
-        language: pygrep
-        entry: '\t'
-        files: '^peps/pep-\d+\.rst$'
-        types: [text]
-
       - id: check-required-headers
         name: "PEPs must have all required headers"
-        language: pygrep
+        language: 'pygrep'
         entry: '(?-m:^PEP:(?=[\s\S]*\nTitle:)(?=[\s\S]*\nAuthor:)(?=[\s\S]*\nStatus:)(?=[\s\S]*\nType:)(?=[\s\S]*\nContent-Type:)(?=[\s\S]*\nCreated:))'
-        args: ['--negate', '--multiline']
+        args:
+          - '--negate'
+          - '--multiline'
```

Comment:
And skip these too, because they'll hopefully be removed anyway too, and I prefer the more compact version*.

(* Until the line gets longer, and like the Black repo, I tend to use Prettier to autoformat.)